base/docs/basedocs.jl

```diff
@@ -644,6 +644,31 @@ desired can be used.
 """
 kw"if", kw"elseif", kw"else"
 
+"""
+    a ? b : c
+
+Short form for conditionals; read "if `a`, evaluate `b` otherwise evaluate `c`".
+Also known as the
+[ternary operator](https://docs.julialang.org/en/v1/manual/control-flow/#man-conditional-evaluation-1).
+
+This syntax is equivalent to `if a; b else c end`, but is often used to
+emphasize the value `b`-or-`c` which is being used as part of a larger
+expression, rather than the side effects that evaluating `b` or `c` may have.
+
```

Comment:
```suggestion

See the manual section on [control flow](@ref man-conditional-evaluation) for more details.

```

Reply:
Great idea. Unfortunately this doesn't work for the interactive help mode, which is why I didn't do it this way :-/

Reply:
We'd need to add `@ref` handling to interactive help processing code to have it redirect to the manual.  I think this would be great by the way!  But I didn't want to get too distracted.

Reply:
Yes, but that URL will break soon. My suggestion is the way we handle it everywhere else:
```
$ grep -r --include \*.jl "@ref" . | wc -l 
3150
$ grep -r --include \*.jl "docs.julialang.org" . | wc -l 
8
```
The 8 occurences of that URL is in `doc/make.jl` and in the code for the banner, so there are actually no URLs in docstrings.

Reply:
Ok let's do it that way. Having the `@ref` work in help mode will have to wait for another time.